python_modules/libraries/dagster-azure/dagster_azure/adls2/io_manager.py

```diff
@@ -39,6 +41,10 @@ def _rm_object(self, key):
         check.param_invariant(len(key) > 0, "key")
 
         # This operates recursively already so is nice and simple.
```

Comment:
this comment looks like it corresponds to the final line in the function, not this one